tests/test_geoprocessing.py

```diff
@@ -285,8 +294,314 @@ def test_reproject_vector(self):
+        pygeoprocessing.reproject_vector(
+            base_vector_path, target_reference.ExportToWkt(),
+            target_vector_path, layer_id=0, copy_fields=['id'])
+
+        vector = ogr.Open(target_vector_path)
+        layer = vector.GetLayer()
+        result_reference = layer.GetSpatialRef()
+        layer_defn = layer.GetLayerDefn()
+        self.assertTrue(
+            osr.SpatialReference(result_reference.ExportToWkt()).IsSame(
+                osr.SpatialReference(target_reference.ExportToWkt())))
+        self.assertTrue(layer_defn.GetFieldCount(), 1)
+        self.assertEqual(layer_defn.GetFieldIndex('id'), 0)
+
+        target_vector_no_fields_path = os.path.join(
+            self.workspace_dir, 'target_vector_no_fields.shp')
+        pygeoprocessing.reproject_vector(
+            base_vector_path, target_reference.ExportToWkt(),
+            target_vector_no_fields_path, layer_id=0, copy_fields=False)
+        layer = None
+        vector = None
+
+        vector = ogr.Open(target_vector_no_fields_path)
+        layer = vector.GetLayer()
+        result_reference = layer.GetSpatialRef()
+        layer_defn = layer.GetLayerDefn()
+        self.assertTrue(
+            osr.SpatialReference(result_reference.ExportToWkt()).IsSame(
+                osr.SpatialReference(target_reference.ExportToWkt())))
+        self.assertTrue(layer_defn.GetFieldCount(), 0)
         layer = None
         vector = None
+
+    def test_reproject_vector_latlon_to_utm(self):
+        """PGP.geoprocessing: reproject vector from lat/lon to utm."""
+        import pygeoprocessing
+        # Create polygon shapefile to reproject
+        base_srs = osr.SpatialReference()
+        base_srs.ImportFromEPSG(4326) # WGS84
+        extents = [-123.71107369, 44.7600990, -121.71107369, 43.7600990]
+
+        polygon_a = shapely.geometry.Polygon([
+            (extents[0], extents[3]), (extents[0], extents[1]),
+            (extents[2], extents[1]), (extents[2], extents[3]),
+            (extents[0], extents[3])])
+
+        base_vector_path = os.path.join(
+            self.workspace_dir, 'base_vector.shp')
+        field_name = 'id'
+        test_driver = ogr.GetDriverByName('ESRI Shapefile')
+        test_vector = test_driver.CreateDataSource(base_vector_path)
+        test_layer = test_vector.CreateLayer('base_layer', srs=base_srs)
+
+        field_defn = ogr.FieldDefn(field_name, ogr.OFTInteger)
+        test_layer.CreateField(field_defn)
+        layer_defn = test_layer.GetLayerDefn()
+
+        test_feature = ogr.Feature(layer_defn)
+        test_geometry = ogr.CreateGeometryFromWkb(polygon_a.wkb)
+        test_feature.SetGeometry(test_geometry)
+        test_feature.SetField(field_name, 0)
+        test_layer.CreateFeature(test_feature)
+
+        test_layer = None
+        test_vector = None
+        test_driver = None
+
+        target_reference = osr.SpatialReference()
+        target_reference.ImportFromEPSG(3157) # UTM zone 10N

```

Comment:
There's another `PEP8` instance here for a comment.  There might be others here in this file as well ... could you try running `pycodestyle` (`pip install pycodestyle`) to check?

Reply:
Done here: [f2c337b](https://github.com/natcap/pygeoprocessing/pull/31/commits/f2c337bce218e8c7f7d922edf0eb32f4d962f6df)
Thanks for the `pycodestyle` tool!